src/movies.js

```diff
@@ -1,16 +1,45 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
+ 
+const getAllDirectors = movies.map(function (directors) {
+    return directors.director;
```

Comment:
The indentation seems off here.
Please be aware about tabs/horizontal formatting.